xml/System.Text.Json/JsonTokenType.xml

```diff
@@ -134,7 +145,7 @@
       </ReturnValue>
       <MemberValue>10</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The token type is the JSON literal <see langword="null" />.</summary>
```

Comment:
Same issue here. This isn't the .NET `null`.

Reply:
Let's use "null" then:

```suggestion
        <summary>The token type is the JSON literal "null".</summary>
```